hack/make.ps1

```diff
@@ -126,6 +126,25 @@ Function Check-InContainer() {
         Write-Host ""
         Write-Warning "Not running in a container. The result might be an incorrect build."
         Write-Host ""
+        return $False
+    }
+    return $True
+}
+
+# Utility function to warn if the version of go is correct. Used for local builds
+# outside of a container where it may be out of date with master.
+Function Verify-GoVersion() {
+    Try {
+        $goVersionDockerfile=(Get-Content ".\Dockerfile" | Select-String "ENV GO_VERSION").ToString().Split(" ")[2]
```

Comment:
Ah no, this is very explicit to check `Dockerfile`, not `Dockerfile.Windows`. I've seen several cases in the past where `Dockerfile` gets updated (being a largely Linux based project, at least historically) and `Dockerfile.Windows` was left behind.